.circleci/config.yml

```diff
@@ -23,6 +24,11 @@ jobs:
             source ~/venv/bin/activate
             cp sample_shavar_list_creation.ini shavar_list_creation.ini
             mkdir test-results
-            python -m pytest -v --junitxml=test-results/junit.xml
+            python -m pytest -v --junitxml=test-results/junit.xml --cov=. --cov-report=html --cov-branch
+            coverage-badge -o coverage.svg
```

Comment:
Can we show the coverage report on terminal as well? You should be able to add `--cov-report=term` and it should display the result on the terminal.

Reply:
Good idea. I pushed a commit that does this.